code-challenges.js

```diff
@@ -9,9 +9,46 @@
 
 // Create the code and test each of the variables provided. Expected output: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-var temp = 35
-// var temp = 350
-// var temp = 212
+// I'm renaming the variable 'temp' to 'temperature' for clarity and readability.
+var temperature = 35
+// var temperature = 350
+// var temperature = 212
+
+// I can write a basic 'if-else' statement that will compare the current value with the number 212, and return the expected output based on wether the current value is greater or less than 212. This works. There's probably a cleaner way though.
+
+if (temperature == 212) {
+    console.log("212 is boiling point")
+} else if (temperature == 350) {
+    console.log("350 is above boiling point")
+} else if (temperature == 35) {
+    console.log("35 is below boiling point")
+} else {
+    console.log("Those are your only choices")
+}
+
+// This version doesn't work yet, and is probably more complicated than it needs to be. But it would be fun to get it working.
+
+// var temperature = {
+//     boiling: 212,
+//     above: 350,
+//     below: 35
+// }
+
+// var message = `${temperature.boiling} is boiling point`
+// var message = `${temperature.above} is above boiling point`
+// var message = `${temperature.below} is below boiling point`
+
+// if (temperature.above === 350) {
+//     console.log(`${temperature.above} is above boiling point`)
+// } else if (temperature.below === 35) {
+//     console.log(`${temperature.below} is below boiling point`)
+// } else {
+//     console.log(`${temperature.boiling} is boiling point`)
+// }
+
+// console.log(message)
```

Comment:
Cool stuff, one reason this isn't working is because message is defined three times.